guides/Getting-Started.md

```diff
@@ -258,15 +260,41 @@ Spago can be used to turn our PureScript code into JavaScript suitable for use i
     Build succeeded.
     Bundle succeeded and output file to index.js
 
-All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document. If you try this, you should see the words "Hello, World!" printed to your browser's console.
+All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document.  Try this by creating the following `index.html` file in your `my-project` directory:
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Euler Exercise</title>
+</head>
+
+<body>
+  <div id="container"></div>
+  <script src="./index.js"></script>
+</body>
+
+</html>
+```
+
+See this HTML file work as follows:
+
+1.  Run `npx parcel index.html --open` in a terminal opened to your `my-project`.
+1.  In the newly-opened browser, browse `localhost:1234`.  You will probably see a blank window.
+1.  Open a **debugger console**.
+1.  Observe the console contains the Euler exercise result string `The answer is 233168`.
 
 If you open `index.js`, you should see a few compiled modules which look like this:
 
+#### How Purescript Compiler Generates Javascript
```

Comment:
Nit: It's "PureScript", not "Purescript". Could you move this header up above the "If you open index.js" line too please?

Reply:
@hdgarrood there were two instances of `Purescript` in this doc; updated both.  Good catch on moving `index.js`.

Reply:
Thanks! Oh sorry I also just spotted this, could you add a “the” before “PureScript” too please?

Reply:
Yep, done (and you'll see it when I upload it).